index.html

```diff
@@ -177,7 +177,47 @@ <h2>Multi-Post Stories</h2>
                 <button class="btn-1">See Project</button>
               </div>
             </div>
+          </div>
+        </div>
+          <div class="sub-grid-4">
+            <div class="sub-flex-2">
+              <div class="box">
+                <input type="text">
+              </div>
+              <div class="story-title">
+                <div>
+                  <h2>Multi-Post Stories</h2>
+                  <p>A daily selection of privately<br> personalized read; no accounts<br> or sign-ups required, has been
+                  the<br> industry's standard dummy text<br> ever since the 1500s, when an<br> unknown printer took a
+                  took a standard<br> dummy text</p>
+                </div>
+                <div>
+                  <ul>
+                    <li>css</li>
+                    <li>html</li>
+                    <li>bootstrap</li>
+                    <li>Ruby</li>
+                  </ul>
+                </div>
+                <div>
+                  <button class="btn-1">See Project</button>
+                </div>
+              </div>
+            </div>
+          </div>
       </div>
+            <div class="sub-grid-5">
+              <h2 class="text-align">CONTACT US</h2>
+              <p class="text-align">If you have an application you are<br> interested in developing, a feature that you<br> need
+                built or a project that needs coding.<br> I’d love to help with it</p>
+                <form class="text-align" action="/Ademola-Portfolio" method="post">
+                  <ul>
+                    <li><input type="text" id="text_input" name="text_input" placeholder="Text input"></li>
+                    <li><input type="text" id="email" name="email" placeholder="scdk@yahoo.com"></li>
+                  </ul>
+                  <textarea name="user_message" id="msg" cols="30" rows="10">Enter text here</textarea>
```

Comment:
- [ ] Message input should have `maxlength="500"` attribute and must be required